examples/SLAM/js/index.ts

```diff
@@ -0,0 +1,11 @@
+
+import '@wonderlandengine/components';
+
+import './hit-test-location-root';
+import './hit-test-location-xr8';
+import './spawn-mesh-on-reticle';
+
+import '../../common-components/';
+
+import { ARSLAMCamera } from '../../../src/components/AR/cameras/AR-SLAM-camera';
+WL.registerComponent(ARSLAMCamera);
```

Comment:
With `autoManageImports`, you can replace it with:
```js
/**
 * /!\ This file is auto-generated.
 *
 * This is the entry point of your standalone application.
 *
 * You should register the component you need here, e.g.,
 *
 * ```
 * import { MyComponent } from './my-component.js';
 *
 * WL.registerComponent(MyComponent);
 * ```
 *
 * The `wle:auto-imports:start` and `wle:auto-imports:end` comments are
 * used by the editor as a target for the import list.
 */

/* wle:auto-imports:start */
/* wle:auto-imports:end */
```

Reply:
If WL.registerComponent can happen only in the index.js file, how would the ARSLAMCamera get registered?
I intentionally want to import camera from `import { ARSLAMCamera } from '@wonderlandengine/ar-tracking/components/AR/cameras/AR-SLAM-camera';` instead of `import { ARSLAMCamera } from '@wonderlandengine/ar-tracking'`, since then the editor does not have to load all the available cameras.
On the other hand, the default components are loaded all at once, so maybe I can load all the cameras by default as well to keep the convention.

Reply:
Why do you need to register it? The user is in charge of doing it. By default, the user doesn't need to because we have a system where all components found in `node_modules` are pre-registered.